INSTALL.md

```diff
@@ -69,17 +77,31 @@ If you are a Node.js developer, you may wish to install the balena CLI via [npm]
 The npm installation involves building native (platform-specific) binary modules, which require
 some additional development tools to be installed first:
 
-* Node.js version 8 or 10 (v12 has not been thoroughly tested yet)
+* Node.js version 8, 10 or 12 (on Linux/Mac, [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md)
+  is recommended)
 * Python 2.7
 * g++ compiler
 * make
 * git
-* On Windows, the `windows-build-tools` npm package should be installed too, running the following
-  command in an administrator console (available as "Command Prompt (Admin)" or "Windows PowerShell
-  (Admin)" when typing Windows+X):  
+
+On Windows, the dependencies above and additional ones can be met with:
+
+* The [MSYS2 shell](https://www.msys2.org/) may be used to provide `git`, `ssh`, `rsync`, `make`
+  and `g++`:
+  * `pacman -S git openssh rsync gcc make`
+  * [Set a Windows environment variable](https://www.onmsft.com/how-to/how-to-set-an-environment-variable-in-windows-10): `MSYS2_PATH_TYPE=inherit`
+  * Note that a bug in the MSYS2 launch script (`msys2_shell.cmd`) makes text-based
+    interactive CLI menus to misbehave. [Check this Github issue for a
+    workaround](https://github.com/msys2/MINGW-packages/issues/1633#issuecomment-240583890).
+* Install the Windows Driver Kit (WDK) which is needed to compile some native Node modules:
+  * [WDK for Windows 10](https://docs.microsoft.com/en-us/windows-hardware/drivers/download-the-wdk)
+  * [WDK for earlier versions of Windows](https://docs.microsoft.com/en-us/windows-hardware/drivers/other-wdk-downloads)
```

Comment:
The WDK installation step addresses an error reported in the forums, which I had also recently observed myself in a fresh Windows 10 installation:
https://forums.balena.io/t/balena-cli-install-err/5606
```
embedder : error : Could not open file 'C:\Program Files (x86)\Windows Kits\10\redist\wdf\x64\WdfCoInstaller01011.dll'. [C:\Users\confusetador\AppData\Roaming\npm\node_modules\balena-cli\node_modules\winusb-driver-generator\build\libwdi.vcxproj]
    EMBED  C:\Program Files (x86)\Windows Kits\10\redist\wdf\x64\WdfCoInstaller01011.dll
gyp ERR! build error
```